backfill/2024-04-26-active_users_aggregates/desktop_schema.yaml

```diff
@@ -0,0 +1,79 @@
+fields:
+- name: segment
+  type: STRING
+  mode: NULLABLE
+- name: app_name
+  type: STRING
+  mode: NULLABLE
```

Comment:
Not blocking, but a general question -- does it make sense for this to ever be NULL? Or is every column just `NULLABLE` out of convenience?

Reply:
Mostly out of convenience.